web/src/components/dashboard/SidebarCardGeneric.tsx

```diff
@@ -1,106 +1,99 @@
-  );
-
-  const rowBodyContent = (
-    <div className="flex fdr fac">
-      <Content className="margin-right-1">{props.bodyText}</Content>
-      {props.ctaOnClick && props.card.ctaText && <div className="width-100">{ctaButton}</div>}
-    </div>
+    </PrimaryButton>
   );
 
   return (
     <>
       <div
-        className={`border radius-md border-${props.card.borderColor} margin-bottom-3`}
+        className={`border radius-md sideBarCardBorder margin-bottom-3 ${sideCardClassDictionary.get(
+          props.card.id
+        )}`}
         {...{ "data-testid": props.card.id }}
       >
         {props.headerText && (
-          <div
-            className={`bg-${props.card.headerBackgroundColor} padding-y-105 padding-x-205 ${
-              props.card.headerBackgroundColor === "white" ? "padding-top-105 padding-bottom-105" : ""
-            } radius-top-md`}
-          >
+          <div className="padding-left-205 radius-top-md" style={{ paddingTop: ".75rem" }}>
             <div className="flex flex-justify">
-              <h3
-                className={`margin-bottom-0 text-${props.card.color} ${
-                  props.card.imgPath ? "flex flex-align-start" : ""
-                }`}
-              >
-                {props.card.imgPath && (
-                  <img
-                    role="presentation"
-                    className="margin-right-2 height-4 width-4"
-                    src={props.card.imgPath}
-                    alt=""
-                  />
-                )}
-                <span className={props.card.imgPath ? "margin-top-2px padding-top-1px" : ""}>
+              <h3 className={`margin-bottom-0 text-white`}>
+                <span>
                   <ModifiedContent>{props.headerText}</ModifiedContent>
                 </span>
               </h3>
               {props.card.hasCloseButton && (
                 <UnStyledButton style="default" onClick={closeSelf} isAriaLabelApplied="Close">
-                  <Icon className={`font-sans-xl text-${props.card.color}`}>close</Icon>
+                  <Icon className={`font-sans-xl text-white`}>close</Icon>
                 </UnStyledButton>
               )}
             </div>
           </div>
         )}
         <div
-          className={`bg-white padding-205 text-base radius-bottom-md ${
+          className={`padding-205 text-base radius-bottom-md text-white ${
             !props.card.header && "radius-top-md"
           }`}
         >
-          {props.preBodyContent && <div className={`padding-bottom-205`}>{props.preBodyContent}</div>}
-          {layout === "row" && !isMobile ? rowBodyContent : stackedBodyContent}
+          {props.preBodySpanButtonText && (
+            <div style={{ float: "left" }}>
+              <UnStyledButton
+                style="default"
+                isTextBold
+                onClick={props.preBodyButtonOnClick}
+                dataTestid="link-to-profile-card"
+                isAriaLabelApplied="Go to your profile"
```

Comment:
pebble, this is a component that renders across cards, `dataTestid` and `isAriaLabelApplied` need to be dynamic. If content added `preBodySpanButtonText` to another card, the label / test id shouldn't be `Go to your profile`/ `link-to-profile-card`

Reply:
why do we need isAriaLabelApplied?

Reply:
done